adapters/AppLovin/AppLovinAdapter/RTB/AppLovinRtbAdapter.m

```diff
@@ -0,0 +1,137 @@
+//
+//  AppLovinRtbAdapter.m
+//  Adapter
+//
+//  Created by Christopher Cong on 7/17/18.
+//  Copyright © 2018 Google. All rights reserved.
+//
+
+#import "AppLovinRtbAdapter.h"
+#import "GADMAdapterAppLovinExtras.h"
+#import "GADMAdapterAppLovinUtils.h"
+#import "GADMAdapterAppLovinConstant.h"
+#import "GADMAppLovinRtbBannerRenderer.h"
+#import "GADMAppLovinRtbInterstitialRenderer.h"
+#import "GADMAppLovinRtbRewardedRenderer.h"
+
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface AppLovinRtbAdapter ()
+// Controlled Properties
+@property (nonatomic, copy) GADMediationServerConfiguration *configuration;
+@property (nonatomic, copy) GADRTBSignalCompletionHandler signalCompletionHandler;
+
+@property (nonatomic, strong) GADMAppLovinRtbBannerRenderer *bannerRenderer;
+@property (nonatomic, strong) GADMAppLovinRtbInterstitialRenderer *interstitialRenderer;
+@property (nonatomic, strong) GADMAppLovinRtbRewardedRenderer *rewardedRenderer;
+@end
+
+@implementation AppLovinRtbAdapter
+
+static NSString *const kAppLovinRtbAdapterErrorDomain = @"com.applovin.AppLovinRtbAdapter";
```

Comment:
Leftover code?

Reply:
We use this as the error domain for the RTB adapter. I'll update the name move to Utils.